src/config/jest-config-resolver.ts

```diff
@@ -0,0 +1,47 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import * as resolve from 'resolve'
```

Comment:
I believe that related to different module import modes, you should here use granulated imports